qos-service-impl/src/main/java/com/palantir/atlasdb/qos/config/ThrottlingStrategy.java

```diff
@@ -20,6 +20,5 @@
 import java.util.function.Supplier;
 
 public interface ThrottlingStrategy {
-    double getClientLimitMultiplier(Supplier<List<CassandraHealthMetricMeasurement>> metricMeasurements,
-            QosPriority qosPriority);
+    double getClientLimitMultiplier(Supplier<List<CassandraHealthMetricMeasurement>> metricMeasurements);
```

Comment:
Plan is to branch on QoSPriority before calling here?

Reply:
Not entirely, removed it as it was hanging around unused at least in this version. We can have the branching at either places, I would prefer it in here. We can do more complicated things for the `LOW` or `MEDIUM` priorities but not in the first version. for eg: we can further scale down a `LOW` priority and scale up a `MEDIUM` priority and keep track of them so that (num clients scaled up ~ num clients scaled down). But that should be a part of a separate PR as we move forward with testing.